mythril/laser/ethereum/state/world_state.py

```diff
@@ -22,18 +24,29 @@ def __init__(
         :param transaction_sequence:
         :param annotations:
         """
-        self.accounts = {}  # type: Dict[str, Account]
+        self._accounts = {}  # type: Dict[int, Account]
+        self.balance = Array("balance", 256, 256)
```

Comment:
Any reason why this is singular? (self.balance over self.balances)

Reply:
not really, I changed it to be plural 👍